app/views/casework/crime_applications/sections/_properties.html.erb

```diff
@@ -0,0 +1,141 @@
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            <%= label_text(:size_in_acres, scope: 'properties') %>
+          </dt>
+          <dd class="govuk-summary-list__value">
+            <%= property.size_in_acres %>
+          </dd>
+        </div>
+        <% end %>
+        <% if property.property_type == PropertiesPresenter::LAND || property.property_type == PropertiesPresenter::COMMERCIAL %>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            <%= label_text(:usage, scope: 'properties', asset: asset) %>
+          </dt>
+          <dd class="govuk-summary-list__value">
+            <%= property.usage %>
+          </dd>
+        </div>
+        <% end %>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            <%= label_text(:value, scope: 'properties', asset: asset) %>
+          </dt>
+          <dd class="govuk-summary-list__value">
+            <%= number_to_currency(property.value / 100, unit: '£', separator: '.', delimiter: ',') %>
+          </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            <%= label_text(:outstanding_mortgage, scope: 'properties') %>
+          </dt>
+          <dd class="govuk-summary-list__value">
+            <%= number_to_currency(property.outstanding_mortgage / 100, unit: '£', separator: '.', delimiter: ',') %>
+          </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            <%= label_text(:percentage_applicant_owned, scope: 'properties', asset: asset) %>
+          </dt>
+          <dd class="govuk-summary-list__value">
+            <%= number_to_percentage(property.percentage_applicant_owned, precision: 2) %>
+          </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            <%= label_text(:percentage_partner_owned, scope: 'properties', asset: asset) %>
+          </dt>
+          <dd class="govuk-summary-list__value">
+            <%= number_to_percentage(property.percentage_partner_owned, precision: 2) %>
+          </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            <%= label_text(:is_home_address, scope: 'properties', asset: asset) %>
+          </dt>
+          <dd class="govuk-summary-list__value">
+            <%= t(property.is_home_address, scope: 'values') %>
+          </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            <%= label_text(:address, scope: 'properties', asset: asset) %>
+          </dt>
+          <dd class="govuk-summary-list__value">
+            <%= render 'address', address: property.address %>
+          </dd>
+        </div>
```

Comment:
I'm getting an error viewing the address if I selected yes to the address being the same as the client's address. I think this needs to display the client's address if there is no property address

Reply:
Displaying client's address can be misleading. Instead we should show address only if client filled the address form in property journey